### PR TITLE
eip155-877.json

### PR DESCRIPTION
**Pull request by Dexit-Network**
please add Dexit mainnet RPC

### CODE REVIEW

File: _data/chains/eip155-877.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Dexit Mainnet",
+  "chain": "Dexit",
+  "rpc": [
+    "https://dxt.dexit.network",
+    "https://rpc.dexit.network",
+
```
Comment (ligi) [inline _data/chains/eip155-877.json:7]:
```suggestion
    "https://rpc.dexit.network"
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

also:  The Icon DXT does not exist - was used in eip155-877.json

**Comment by ligi** [comment]
?

**Comment by Dexit-Network** [comment]
hi can you please advise what need to be done if possible can you please help me

**Review by nutty323 — APPROVED** [review]
Approved

**Review by Dexit-Network — COMMENTED** [review]
this information is correct

**Review by nutty323 — APPROVED** [review]

**Review by nutty323 — APPROVED** [review]

**Review by pranver — APPROVED** [review]

**Pull request closed without merging**